bucketeer/src/test/kotlin/io/bucketeer/sdk/android/internal/event/EventCreatorsKtTest.kt

```diff
@@ -0,0 +1,20 @@
+package io.bucketeer.sdk.android.internal.event
+
+import org.junit.Test
+
+class EventCreatorsKtTest {
+  @Test
+  fun testLabelTimeoutValueShouldInDoubleFormat() {
+    val input = 15000L
+    val expectedDouble = 15.0
+    val timeoutString = input.toStringInDoubleFormat()
+    assert(timeoutString == expectedDouble.toString())
+    assert(timeoutString == "15.0")
+
+    val input2 = 1512334557L
```

Comment:
Please add or change the input to less than one second. E.g., 500L.

Reply:
updated in https://github.com/bucketeer-io/android-client-sdk/pull/114/commits/e51a409ba7d4664f9b217d2d84e6f4968b985a52